lib/widgets/login.dart

```diff
@@ -677,17 +677,21 @@ class OrDivider extends StatelessWidget {
 
     return Padding(
       padding: const EdgeInsets.symmetric(vertical: 10),
-      child: Row(crossAxisAlignment: CrossAxisAlignment.center, children: [
-        divider,
-        Padding(
-          padding: const EdgeInsets.symmetric(horizontal: 5),
-          child: Text(zulipLocalizations.loginMethodDivider,
-            textAlign: TextAlign.center,
-            style: TextStyle(
-              color: designVariables.loginOrDividerText,
-              height: 1.5,
-            ).merge(weightVariableTextStyle(context, wght: 600)))),
-        divider,
-      ]));
+      child: Semantics(
+        readOnly: true,
```

Comment:
Why `readOnly: true`? The thing we're describing is not a text field. From the [doc](https://api.flutter.dev/flutter/semantics/SemanticsProperties/readOnly.html):

```dart
  /// If non-null, indicates that this subtree is read only.
  ///
  /// Only applicable when [textField] is true.
  ///
  /// TalkBack/VoiceOver will treat it as non-editable text field.
  final bool? readOnly;
```